Formula/a/astro.rb

```diff
@@ -32,9 +35,12 @@ def install
     version_output = shell_output("#{bin}/astro version")
     assert_match("Astro CLI Version: #{version}", version_output)
 
-    run_output = shell_output("echo 'y' | #{bin}/astro dev init")
-    assert_match(/^Initializing Astro project*/, run_output)
-    assert_predicate testpath/".astro/config.yaml", :exist?
+    mkdir testpath/"astro-project"
+    cd testpath/"astro-project" do
+      run_output = shell_output("#{bin}/astro dev init")
+      assert_match(/^Initialized empty Astro project*/, run_output)
```

Comment:
does not have to be regex call
```suggestion
      assert_match "Initialized empty Astro project", run_output
```

Reply:
thanks for the recommendation, updated the logic